presto-main/src/test/java/io/prestosql/sql/planner/planprinter/TestWindowOperatorStats.java

```diff
@@ -0,0 +1,26 @@
+package io.prestosql.sql.planner.planprinter;
+
+import io.prestosql.operator.WindowInfo;
+import org.testng.annotations.Test;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestWindowOperatorStats
+{
+    @Test
+    public void testWindowOperatorStatsForWindowInfoWithEmptyDriverInfosList()
+    {
+        WindowInfo info = new WindowInfo(emptyList());
```

Comment:
nit:

```suggestion
        WindowInfo info = new WindowInfo(ImmutableList.of());
```

Reply:
emptyList seems a bit more intention revealing in this scenario, as this is a test code.